// zigbee-clusters/lib/clusters/metering.js
'use strict';

const Cluster = require('../Cluster');
const { ZCLDataTypes } = require('../zclTypes');

const ATTRIBUTES = {
  serialNumber: { id: 0x0308, type: ZCLDataTypes.string },
  pricePeriod: { id: 0x0020, type: ZCLDataTypes.enum8({
      'TH..': 0,
      'HC..': 1,
      'HP..': 2,
      'HN..': 4,
      'PM..': 5,
      'HCJB': 6,
      'HCJW': 7,
      'HCJR': 8,
      'HPJB': 9,
      'HPJW': 10,
    }),
  },
  currentSummationDelivered: { id: 0, type: ZCLDataTypes.uint48 },
  currentSummationDeliveredHCHC: { id: 0x0100, type: ZCLDataTypes.uint48 },
  currentSummationDeliveredHCHP: { id: 0x0102, type: ZCLDataTypes.uint48 },
  currentSummationReceived: { id: 1, type: ZCLDataTypes.uint48 },
  currentMaxDemandDelivered: { id: 2, type: ZCLDataTypes.uint48 },
  currentMaxDemandReceived: { id: 3, type: ZCLDataTypes.uint48 },
  powerFactor: { id: 6, type: ZCLDataTypes.int8 },
  multiplier: { id: 769, type: ZCLDataTypes.uint24 },
  divisor: { id: 770, type: ZCLDataTypes.uint24 },
  instantaneousDemand: { id: 1024, type: ZCLDataTypes.int24 },
};

const COMMANDS = {};

class MeteringCluster extends Cluster {

  static get ID() {
    return 1794; // 0x0702
  }

  static get NAME() {
    return 'metering';
  }

  static get ATTRIBUTES() {
    return ATTRIBUTES;
  }

  static get COMMANDS() {
    return COMMANDS;
  }

}

Cluster.addCluster(MeteringCluster);

module.exports = MeteringCluster;
