diff --git a/node_modules/zigbee-clusters/lib/zclFrames.js b/node_modules/zigbee-clusters/lib/zclFrames.js
index 9275cb6..fc3dde7 100644
--- a/node_modules/zigbee-clusters/lib/zclFrames.js
+++ b/node_modules/zigbee-clusters/lib/zclFrames.js
@@ -60,7 +60,7 @@ const ZCLAttributeDataRecord = (withStatus, attributes) => new ZCLDataType(NaN,
       const dataTypeId = ZCLDataTypes.uint8.fromBuffer(buf, i);
       i += ZCLDataTypes.uint8.length;
 
-      const DataType = attributes[res.id]
+      const DataType = attributes[res.id] && dataTypeId !== 0 //don't use the attribute's type if the type in the data is 0 (noData). This breaks when the attribute ID is 0
         ? attributes[res.id].type
         : Object.values(ZCLDataTypes).find(type => type && type.id === dataTypeId);
       if (!DataType) throw new TypeError(`Invalid Type for Attribute: ${res.id}`);
